app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  def page_title(page:)
+    page_title_translation_key = "page_titles.#{page}"
+
+    if I18n.exists?(page_title_translation_key, :en)
+      "#{t(page_title_translation_key)} - #{t('page_titles.application')}"
+    else
+      t('page_titles.application')
+    end
+  end
```

Comment:
You might want to replace the `if`, something like:  (up to you)
```
title = t('page_titles.application').present? ? t('page_titles.application') :  ''
t(page_title_translation_key) + title
```

Reply:
I've decided to keep it how it is as I like that it's really simple and easy to read.

Reply:
cool, thanks for removing `:en`